doc/user/unit-testing.md

```diff
@@ -1,132 +1,257 @@
-# Unit testing
+<!-- Link vars declaration -->
+[doc_e2e_test]:./test-framework/writing-e2e-tests.md
```

Comment:
Fix broken reference
```suggestion
[doc_e2e_test]:./../test-framework/writing-e2e-tests.md
```

Reply:
It did not pass in the tests as well. :-(

Reply:
@LiliC and @hasbro17 relative paths are not passing in the test so I keep it as it was.